raiden-dapp/package.json

```diff
@@ -6,14 +6,14 @@
   "author": "brainbot labs est.",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "serve:no-logging": "export NODE_ENV=nologging && vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e",
+    "test:e2e": "nyc vue-cli-service test:e2e --mode e2e",
```

Comment:
Do you think making it `--headless` and including it in the nightly test run makes sense?
Probably not at the moment because there is not a lot tested but maybe in the future

Reply:
Absolutely. As soon as we have more tests. Maybe depending on the runtime we could also have it in the regular workflow parallel to the other tests. Basically I was thinking the same for the sdk integration. The current runtime is around 3-4minutes so if we make it parallel to the other tests it should not increase the overall execution time.